projects/m2/023-scrabble-score/js/index.js

```diff
@@ -0,0 +1,30 @@
+const points = {
+    1: ["A", "E", "I", "L", "N", "O", "R", "S", "T", "U"],
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: "K",
+    8: ["J", "X"],
+    10: ["Q", "Z"]
+}
+
+const string = "hello"
+
+
+function scrabblePoint(p, s) {
+    let score = 0
+    for (i = 0; i < s.length; i++) {
```

Comment:
Remember to use the `let` statement inside the for construct (see also other exercises).
`for (let i = 0 .......`